src/Datadog.Trace/Datadog.Trace.nuspec

```
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<package  xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata>    
+    <id>Datadog.Trace</id>
+    <version>1.7.0</version>
+    <title>Datadog APM</title>
+    <authors>lucas.pimentel.datadog;colinhigginsdatadog;zachmontoyadd</authors>
+    <owners>Datadog</owners>
+    <licenseUrl>https://github.com/DataDog/dd-trace-dotnet/blob/master/LICENSE</licenseUrl>
```

`licenseUrl` is deprecated. See [`src/Directory.Build.props`](https://github.com/DataDog/dd-trace-dotnet/blob/master/src/Directory.Build.props).
```suggestion
    <license>Apache-2.0</license>
```